AdminWebsite/AdminWebsite/Services/HearingsService.cs

```diff
@@ -107,7 +107,7 @@ public List<EditParticipantRequest> GetAddedParticipant(List<EditParticipantRequ
         {
             // Add a new participant
             // Map the request except the username
-            if (participant.CaseRoleName == RoleNames.Judge || (_featureToggles.UseV2Api() && participant.HearingRoleName is RoleNames.PanelMember or RoleNames.Winger))
+            if (participant.CaseRoleName == RoleNames.Judge || (participant.HearingRoleName is RoleNames.PanelMember or RoleNames.Winger))
```

Comment:
Please remove IFeatureToggles from the constructor and in the test setup

Reply:
done